From 071fb3df847e0e88f5b111e3d8ee6f8e40a6e01b Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Tue, 13 Nov 2018 20:18:41 -0700
Subject: [PATCH 2/5] msm8974-common: sepolicy: Resolve mediaserver denials

* avc: denied { read } for name="/" dev="tmpfs" ino=6954
  scontext=u:r:mediaserver:s0 tcontext=u:object_r:device:s0
  tclass=dir permissive=1
* avc: denied { open } for name="/" dev="tmpfs" ino=6954
  scontext=u:r:mediaserver:s0 tcontext=u:object_r:device:s0
  tclass=dir permissive=1

Change-Id: I32540b8ec48ff61568094ec87d1bad64570a8e55
---
 sepolicy/common/mediaserver.te | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sepolicy/common/mediaserver.te b/sepolicy/common/mediaserver.te
index 7990af5..774681c 100644
--- a/sepolicy/common/mediaserver.te
+++ b/sepolicy/common/mediaserver.te
@@ -1,4 +1,5 @@
 allow mediaserver camera_socket:sock_file write;
+allow mediaserver device:dir r_dir_perms;
 allow mediaserver hal_camera_hwservice:hwservice_manager find;
 allow mediaserver mm-qcamerad:unix_dgram_socket sendto;
 allow mediaserver sysfs_camera:dir search;
-- 
2.17.1

